LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-model/animation-types/property-types.js

```diff
@@ -200,7 +200,7 @@ const lengthPairType = {
       testAnimationSamples(
         animation,
         idlName,
-        [{ time: 0, expected: '20px 20px' }]
+        [{ time: 0, expected: '20px' }]
```

Comment:
It might need more changes like - https://github.com/web-platform-tests/wpt/commit/b9f5f525800cf1cfbdb46acd2d1bd1d72feb3d11#diff-be868bff8396deaf6494fa52c78c79bb327ac91f88de608413906d48c258b45d

Reply:
^ This would fix your `web-animations` tests.

Reply:
@AlexsanderDamaceno @Ahmad-S792's suggestion would fix your failure, essentially perspective-origin (a different CSS property) ends up also expecting 1 instead of two values, what Ahmad suggested would allow special casing border-spacing.